src/js/migrations.js

```diff
@@ -72,6 +72,71 @@ exports.Migrations= {
     }
   },
 
+  forgetMistakenlyBlockedDomains: function (badger) {
+    let MISTAKES = [
```

Comment:
Can you add a `console.log('RUNNING MIGRATION ...');` like in the above migrations?